BasicSample/Assets/GrabAndThrow/Scripts/GrabAndThrowBehavior.cs

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+using UnityEngine.XR;
+
+namespace Microsoft.MixedReality.OpenXR.BasicSample
+{
+	[RequireComponent(typeof(Rigidbody))]
+	public class GrabAndThrowBehavior : MonoBehaviour
+	{
+		private const float PICKABLE_RANGE = 0.5f;
```

Comment:
would be nice to add comment about how is this range measured.  Is this the distance from the surface of the object to the center of hand (grip pose)?   If so, i would expect 0.05 meter is enough.

Reply:
Added a comment for the new constant in the same place. It has a different use than this constant originally did, but hopefully its new purpose is clear